steps/cli/oc.go

```diff
@@ -106,10 +106,17 @@ var _ = gauge.Step("Update addon config with clusterTasks as <clusterTaskStatus>
 })
 
 var _ = gauge.Step("Create project <projectName>", func(projectName string) {
-	log.Printf("Creating project %v", projectName)
-	oc.CreateNewProject(projectName)
-	store.Clients().NewClientSet(projectName)
-	gauge.GetScenarioStore()["namespace"] = projectName
+	log.Printf("Check if project %v already exists", projectName)
+	if oc.CheckProjectExists(projectName) {
+		log.Printf("Switch to project %v", projectName)
+		store.Clients().NewClientSet(projectName)
+		gauge.GetScenarioStore()["namespace"] = projectName
+	}else {
+		log.Printf("Creating project %v", projectName)
+		oc.CreateNewProject(projectName)
+		store.Clients().NewClientSet(projectName)
+		gauge.GetScenarioStore()["namespace"] = projectName
+	}
```

Comment:
You have repeating code (2 lines) in both branches, you can extract it outside of if/else statement